dev.rollup.config.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-env node */
+'use strict';
+
+import pkg from './package.json';
+import os from 'os';
+import fs from 'fs-extra';
+import path from 'path';
+import glob from 'glob';
+import serve from 'rollup-plugin-serve';
+import livereload from 'rollup-plugin-livereload';
+import { umdConfig } from './rollup.config';
+
+const BROWSER_FILENAME = `${pkg.name}.js`;
+const tmpDestPath = fs.mkdtempSync(path.join(os.tmpdir(), `${pkg.name}-dev-server-`));
+
+const httpsConfig = {
+    key: fs.readFileSync('sslcert-dev/key.pem'),
+    cert: fs.readFileSync('sslcert-dev/cert.pem')
+};
+
+const transformExampleSdkUrl = (buffer) => {
+    return buffer.replace(
+        /(\s*<script.*src=")([^"]+client-app-sdk[^"]+)(".*<\/script>\s*)/i,
+        '$1' + (BROWSER_FILENAME || '$2') + '$3\n'
+    );
+};
+
+const buildExample = (relativeFilePath) => {
+    fs.outputFileSync(
+        path.join(tmpDestPath, relativeFilePath.replace('examples/', '')),
+        transformExampleSdkUrl(fs.readFileSync(relativeFilePath, 'utf8'))
+    );
+};
+
+// Build all examples initially
+glob.sync('examples/**/*', { nodir: true }).forEach(buildExample);
+
+export default Object.assign({}, umdConfig, {
```

Comment:
This is :fire: @jakejrichards!  Turned out great, works perfectly.  Tested watch/live reload for both sdk and examples.